### PR TITLE
Cleanup tabs and tests on My PL page

### PR DESCRIPTION
**Pull request by TurnerRiley**
This PR cleans up how the different [My PL page](https://studio.code.org/my-professional-learning) tabs are rendered so it's more readable and easier to add more content in the future. Aside from adding unit tests to test that the right sections are in each tab, this PR does the following cleanup tasks:
- Show the "Regional Partner Center" tab and the "Workshop Organizer Center" tab if the user has the `program_manager` or the `workshop_organizer` permissions, respectively
- Render the Enrolled Workshops table and the Workshop Feedback banner on the "Facilitator Center", "Regional Partner Center", and "Workshop Organizer Center" tabs
- Make the section code links purple and the other links black with an underline to match the Figma for the Joined Professional Learning Sections table and the Instructor Professional Learning Sections table

### "Facilitator Center" tab
FacilitatorCenter

### "Instructor Center" tab
UniversalInstructorView

### "Regional Partner Center" tab
Regional Partner Center

### "Workshop Organizer Center" tab
Workshop Organizer Center

### Updates the link styling in the Joined Professional Learning Sections table and the Instructor Professional Learning Sections table
Previous Joined PL Sections table:
My PL with sections 2

New Joined PL Sections table:
NewJoinedPL

Previous Instructor PL Sections table:
OldInstructed

New Instructor PL Sections table:
NewInstructedPLSectionsTable

## Links
Jira ticket (general cleanup): [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1795)
Jira ticket (Workshop table on right tabs): [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1823)
Jira ticket (Workshop Feedback section on right tabs): [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1822)
Figma of page for reference: [here](https://www.figma.com/file/sZ6d1uhJWFksJ6j3Fnmbnc/PL-Flow-Improvements?type=design&node-id=1958-10169&mode=design&t=xL85nqt7JZC0Wvd2-0)

## Testing story
Local testing and adding unit tests.

### CODE REVIEW

File: apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx [hunk 1]
```diff
@@ -167,6 +184,87 @@ function LandingPage({
...
+      </>
+    );
+  };
+
+  const RenderFacilitatorCenterTab = () => {
+    return (
+      <>
+        {lastWorkshopSurveyUrl && RenderLastWorkshopSurveyBanner()}
+        {RenderOwnedPlSections()}
+        {RenderEnrolledWorkshops()}
```
Comment (bethanyaconnor) [inline apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx]: For facilitators, regional partners, and workshop organizers, we want to render the workshops related to that role. That data should be passed down to the client, but you might need to pull it through to this component.

[Here's](https://github.com/code-dot-org/code-dot-org/blob/105a338f60c14e399e5febca6c647cfd3a4749f6/dashboard/app/controllers/pd/professional_learning_landing_controller.rb#L24-L31) where that data is fetched in the controller.

File: apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx [hunk 2]
```diff
@@ -184,56 +282,12 @@ function LandingPage({
...
-            <CoteacherInviteNotification isForPl={true} />
-            <OwnedSections
-              isPlSections={true}
-              sectionIds={plSectionIds}
-              hiddenSectionIds={hiddenPlSectionIds}
-            />
-            <AddSectionDialog />
-          </section>
-        )}
+        {currentTab === 'myPL' && RenderMyPlTab()}
```
Comment (bethanyaconnor) [inline apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx:295]: so clean 🤩

**Review by bethanyaconnor — COMMENTED** [review]
Thanks for doing all of this! It looks good, but one comment about workshops.

File: apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx (hunk 1 above)
Comment (TurnerRiley) [reply apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx]: Oh you're so right, my bad! I'll fix this!

File: apps/src/code-studio/pd/professional_learning_landing/EnrolledWorkshops.jsx [hunk 3]
```diff
@@ -209,8 +225,12 @@ class EnrolledWorkshopsTable extends React.Component {
             </Button>
           </Modal.Footer>
         </Modal>
-        <h2>My Workshops</h2>
-        {this.renderWorkshopsTable()}
+        {this.props.workshops && (
+          <section>
```
Comment (kelbyhawn) [inline apps/src/code-studio/pd/professional_learning_landing/EnrolledWorkshops.jsx:229]: 🎉

File: apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx (hunk 2 above)
Comment (kelbyhawn) [reply apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx:295]: ➕ Amazing!

**Review by kelbyhawn — APPROVED** [review]
I mostly looked at the `LandingPage.jsx` changes and styling updates, so approving based off of that, but overall this looks incredible!

**Pull request merged**